### DISCUSSION TITLE
Enquiry on what the 'vn_kv' parameter stands for ?

### DISCUSSION THREAD

**Discussion by Horreyneeyo** [e2nIEE/pandapower#2817] [Q&A]
Hello all,

I am a new user of Pandapower. I would like to seek clarification on what 'vn_kv' means. Is it the nominal voltage or rated voltage. For example in Pandapower.networks.testcases e.g IEEE 14 bus test case, the buses have a norminal voltages of either 69kV, 13.8kV and 18kV as applicable. However, the voltage specified for the buses on pandapower are different and probably adapted with values of 135kV, 0.208 kV and 12kV.
I am confused as regards how this was derived. I would be grateful if explanation can be given or references materials are pointing out to me.

**Comment by AnkurArohi** [comment]
They are rated values
(https://pandapower.readthedocs.io/en/v2.9.0/elements/bus.html)

Is there a big difference in nominal and rated values for buses in the data model that you have?

**Comment by Horreyneeyo** [comment]
Hello Ankur
![Screenshot 2022-07-26 171306](https://user-images.githubusercontent.com/57833908/181057399-92a1532d-46f0-4432-980b-4fecdc52d87b.png)
![Screenshot 2022-07-26 171104](https://user-images.githubusercontent.com/57833908/181057401-848e22a9-b627-4568-8fd8-9453df6ed677.png)


Thanks for the response. 
The black background is the Pandapower (pp) network case while the white back ground is the original IEEE 14 test case. I understand now that the pp is rated bus voltage (vn_kv). How do you calculate rated bus voltage? Secondly if it indeed rated bus voltage, why are the last 6 values of vn_kv in the pp 0.208kV and in the IEEE test case, they are 13.8kV. Shouldn't the rated voltage be higher than the nominal voltage?

**Comment by AnkurArohi** [comment]
I think you are comparing different files .
IEEE 14 is TSO network.

**Comment by Horreyneeyo** [comment]
Well, IEEE 14 is sub-transmission/distribution. I am comparing the network parameters of the original IEEE 14 test case applicable in Matpower and the pp.networks.case14 parameters. I understand there would be differences e.g. Matpower IEEE 14 uses per-unit parameter values while pp uses per km in the case of RXB.
I am wondering how to determine the rated voltage values as used in pp.

**Comment by SteffenMeinecke** [comment]
As described in the [docs](https://pandapower.readthedocs.io/en/develop/networks/power_system_test_cases.html#case-14), in case14 and case57 assumptions must be made since the data origin have no voltage information.
@Horreyneeyo Which is the origin of the `vn_kv` data you showed?
After 5 HV node, a MV/LV node follows and then there are two nodes which are connected to the medium voltage connection of a 3winding transformer. In pp, `vn_kv` of these nodes are 14 and 12 kV. Why do you have 13.8 and 18?

**Comment by Horreyneeyo** [comment]
The origin of the vn_kv (the white background pic as seen above) is from the IEEE 14 bus datasheet as seen [here](https://www.academia.edu/7781632/A_DATA_SHEETS_FOR_IEEE_14_BUS_SYSTEM).  You are correct about the 3-winding transformer connection. the values 13.8kV and 18kV are the base kV as seen in the aforementioned datasheet above.
